test/e2e/tests/add-custom-network.spec.js

```diff
@@ -17,6 +17,232 @@ describe('Custom network', function () {
       },
     ],
   };
+
+  it('should show warning when adding chainId 0x1', async function () {
```

Comment:
Maybe this title should be updated to reflect that the test is for the scenario where a warning is shown first, followed by an error. It's good that we have this case covered, but I hadn't expected that from this description.